src/diffpy/labpdfproc/tests/test_functions.py

```diff
@@ -36,3 +36,21 @@ def test_set_distances_at_angle(inputs, expected):
     actual_distances_sorted = sorted(actual_gs.distances)
     expected_distances_sorted = sorted(expected_distances)
     assert actual_distances_sorted == pytest.approx(expected_distances_sorted, rel=1e-4, abs=1e-6)
+
+
+params3 = [
+    ([1, 3, 1, 45], [1, 0.2431167, 0.2431167, 0.135335, 0.135335]),
+    ([1, 3, 1, 90], [1, 1, 0.135335, 0.135335, 0.135335]),
+    ([1, 3, 1, 120], [1, 1, 0.135335, 0.049787, 0.176921]),
+]
+
+
+@pytest.mark.parametrize("inputs, expected", params3)
+def test_set_muls_at_angle(inputs, expected):
+    expected_muls = expected
+    actual_gs = Gridded_circle(radius=inputs[0], n_points_on_diameter=inputs[1], mu=inputs[2])
```

Comment:
you are not testing gridded circle so you don't have to give it variable inputs.  Just hard code 1,3,1 in here.